### ISSUE TITLE
VoiceOver does not announce the column headers in the "Add Entity" dialog correctly

### ISSUE THREAD

**Issue by MRayermannMSFT** [Microsoft/AzureStorageExplorer#206]
VoiceOver should be announcing the column header for each control in the table in the Add Entity dialog. See linked bug for more information.

Fixing this will close:
- https://devdiv.visualstudio.com/DevDiv/Storage%20Tools/_workitems/edit/604916

**Comment by MRayermannMSFT** [comment]
Testers not satisfied with the fix (we didn't understand the issue so we didn't actually fix what they think is broken). Reopening the bug, and Craig is going to talk to the testers and the accessibility folks.

**Comment by MRayermannMSFT** [comment]
Testers have closed the bug!
